tools/prepare-static-serving.js

```diff
@@ -0,0 +1,35 @@
+const yaml = require('js-yaml');
+const fs = require('fs');
+
+let doc;
+try {
+  doc = yaml.safeLoad(fs.readFileSync(__dirname + '/../client/flutter/assets/onboarding/iso_countries.en.yaml', 'utf8'));
+  console.log(doc);
+} catch (e) {
+  console.log(e);
+  throw e;
+}
+
+const countries = doc.countries.map((x) => x.alpha_2_code);
+console.log(countries);
+
+const srcPath = __dirname + '/../client/flutter/assets/content_bundles';
+const files = fs.readdirSync(srcPath);
+
+const destDir = __dirname + '/staticContentBuild';
+fs.mkdirSync(destDir);
+
+for (const file of files) {
+  const regex = /^(?<base>.+)\.(?<lang>[^._]+)(?:_(?<country>[^.]+))?\.yaml$/;
+  const m = file.match(regex).groups;
+  console.log(m);
+  const expanded = m.country ? [`${m.base}.${m.lang}_${country}`] : countries.map((c) => `${m.base}.${m.lang}_${c}.yaml`);
```

Comment:
ah, one other note - do we want to do any sort of check for if the filename doesn't match, or are you thinking that throwing an error is the right thing since all files in the directory should match?

Reply:
The js will crash for non-matchng files, which IMO it should.  It'll catch it in the CI.

Reply:
sg, just wanted to make sure it was intentional